{"name": "test_login_failure[chromium-123-admin-Invalid credentials. Please try again.]", "status": "passed", "parameters": [{"name": "browser_name", "value": "'chromium'"}, {"name": "username", "value": "'123'"}, {"name": "password", "value": "'admin'"}, {"name": "result", "value": "'Invalid credentials. Please try again.'"}], "start": 1738128964255, "stop": 1738128969755, "uuid": "f5fe1dda-f2e7-4475-9b5a-22c765dd2a1d", "historyId": "784fbf7dbe03a7b6aabb771d2221e2ee", "testCaseId": "68772be64bb38f6a4c2b59109d7c35bc", "fullName": "test_login#test_login_failure", "labels": [{"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-TR3R9M3"}, {"name": "thread", "value": "45532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}